docs/interface_specification.md

```diff
@@ -152,6 +152,7 @@ A UCE implementation must support the following operations:
 - Handle both uncached and cached requests
 - Support both write-through and write-back protocols
 - Support credit-based flow control, to support fencing in the core
+- Support ordering addresses starting from the requested address (aligned based on the fill width), continuing with the higher addresses and wrapping around to the lower addresses
```

Comment:
This should be in the mem if section, not the UCE section.

"Misaligned addresses return data wrapped around the request size using the following scheme:

Request 0x0: [d c b a]
Request 0x2: [b a d c]"

In this cache service section, we should also clarify that subword requests replicate data to up to dword size

Reply:
Will change that